router/config.go

```diff
@@ -458,6 +484,11 @@ func LoadConfig(path string) (*Config, error) {
 					return nil, errors.Wrap(err, "invalid value for ctrl.defaultRequestTimeout")
 				}
 			}
```

Comment:
For now, if dataDir isn't defined, can we define it as the parent directly of the config file?